citar-file.el

```diff
@@ -77,15 +77,14 @@ will open, via `citar-open-notes'."
 If t, all files whose base name starts with the reference key and
 whose extension is listed in `citar-file-extensions' are located
 by the functions `citar-open-library-files' and
-`citar-open-notes'.  If nil, only files with the naming scheme
-\"<key>.<extension>\" are located.  Otherwise, its value is a
-regular expression specifying how the key is separated from the
-rest of the filename."
+`citar-open-notes'. If nil, only files with the naming scheme
+\"<key>.<extension>\" are located. Otherwise, its value is a
+string to be appended to the key, to separate it from the rest of
+the filename."
   :group 'citar
   :type '(choice (const :tag "Ignore additional files" nil)
                  (const :tag "Find all files starting with key" t)
-                 (const :tag "Find files with space after key" "[[:space:]]")
-                 (regexp :tag "Filename separator")))
+                 (const :tag "Find files with string after key")))
```

Comment:
This doesn't present a text field to add the string in the customize interface though.